app/Http/Controllers/ApiController.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Traits\JsonRespondController;
+
+class ApiController extends Controller
+{
+    use JsonRespondController;
+
+    protected int $limitPerPage = 10;
+
+    protected string $sort = 'created_at';
+
+    protected ?string $withParameter = null;
+
+    protected string $sortDirection = 'asc';
+
+    public function __construct()
+    {
+        $this->middleware(function ($request, $next) {
```

Comment:
Do we really need this now? We never used the 'sort' command before, and I'm not sure the other ones are needed actually.

Reply:
Ok. We use the limit actually, but we don't need the sort right now. So let's remove the rest.